test/models/test_omnivore.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+import torchmultimodal.models.omnivore as omnivore
+from torchmultimodal.utils.common import get_current_device
+
+
+class TestOmnivoreModel(unittest.TestCase):
+    def setUp(self):
+        self.device = get_current_device()
+
+    def test_omnivore_swin_t_forward(self):
+        torch.manual_seed(42)
```

Comment:
Use this:
https://github.com/facebookresearch/multimodal/blob/e8575411d8fdc3f97c51d36789b2f6c530704a50/test/test_utils.py#L60